$NetBSD: patch-al,v 1.3 2009/03/14 11:00:22 apb Exp $

Changes from NetBSD xsrc/external/mit/xorg-server/dist/\
hw/xfree86/xaa/xaalocal.h

----------------------------
revision 1.2
date: 2009/02/19 20:02:02;  author: macallan;  state: Exp;  lines: +11 -0
Actually use scanline image writes to upload images if available
----------------------------

--- hw/xfree86/xaa/xaalocal.h.orig	2009-07-07 17:48:58.000000000 +0000
+++ hw/xfree86/xaa/xaalocal.h
@@ -599,6 +599,17 @@ XAAFillImageWriteRects(
     PixmapPtr pPix
 );
 
+void 
+XAAFillScanlineImageWriteRects(
+    ScrnInfoPtr pScrn,
+    int rop,
+    unsigned int planemask,
+    int nBox,
+    BoxPtr pBox,
+    int xorg, int yorg,
+    PixmapPtr pPix
+);
+
 void
 XAAPolyFillRect(
     DrawablePtr pDraw,
